arch/arm/src/sama5/sam_udphs.c

```diff
@@ -2650,27 +2650,32 @@ static void sam_ep_interrupt(struct sam_usbdev_s *priv, int epno)
           /* Get the size that we expected to receive */
 
           len = GETUINT16(priv->ctrl.len);
-          if (len == pktsize)
+
+          /* Copy the OUT data from the EP0 FIFO into the special EP0 buffer. */
```

Comment:
Please fix the style issue

Reply:
Last minute edit and didn't rerun checkpatch :-o

Sorry! Will sort tomorrow.